src/Input/__tests__/Input.test.js

```diff
@@ -45,4 +51,90 @@ describe('<Input />', () => {
 
     expect(tree).toMatchSnapshot();
   });
+
+  context('with a validate function', () => {
+    const render = props =>
+      mount(
+        <ThemeProvider theme={theme}>
```

Comment:
`mountWithTheme` helper doesn't work here because it returns the component we pass it; we need the top-level wrapper to re-`find` updated elements after state changes.